### PR TITLE
Remove build dependency on `jq` and `apk`

### PR DESCRIPTION
**Pull request by nhinsch**
### What does this PR do?

Previously, we used `jq` during the build process. This build dependency required installation using `apk`.

### Motivation

I have been having a lot of trouble getting `apk` working in my Docker container locally, likely due to some Docker networking issue. Some cursory Googling reveals that this is a common issue. I noticed that `jq` and therefore `apk` were not needed anyway so I decided to remove the dependency.

### Testing Guidelines

The layers still pass the integration tests.

### Types of Changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [x] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [x] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [x] This PR passes the integration tests (ask a Datadog member to run the tests)

### CODE REVIEW

**Comment by nhinsch** [comment]
I realized there is an issue here -- this would install the latest version of dd-trace, which is not necessarily the version specified in the `package.json` as a dev dependency.

**Comment by nhinsch** [comment]
I reopened this PR after implementing a solution that will preserve the old behavior while avoiding use of `jq`.

File: scripts/move_ddtrace_dependency.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Moves the dd-trace dependency from devDependencies to dependencies within package.json.
```
Comment (maxday) [inline scripts/move_ddtrace_dependency.js:1]: 💯

**Review by maxday — APPROVED** [review]
LGTM 🚀

**Pull request merged**